prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -274,57 +272,42 @@ func getReviewers(rc reviewersClient, ghc githubClient, log *logrus.Entry, autho
 		// record required reviewers if any
 		requiredReviewers.Insert(rc.RequiredReviewers(file.Filename).UnsortedList()...)
 
-		fileUnusedLeafs := rc.LeafReviewers(file.Filename).Difference(stage1).Difference(authorSet)
+		fileUnusedLeafs := layeredsets.NewString(rc.LeafReviewers(file.Filename).List()...).Difference(reviewers.Set()).Difference(authorSet)
 		if fileUnusedLeafs.Len() == 0 {
 			continue
 		}
 		leafReviewers = leafReviewers.Union(fileUnusedLeafs)
 		if r := findReviewer(ghc, log, useStatusAvailability, &busyReviewers, &fileUnusedLeafs); r != "" {
-			stage1.Insert(r)
+			reviewers.Insert(0, r)
 		}
 	}
 	// now ensure that we request review from at least minReviewers reviewers. Favor leaf reviewers.
-	unusedLeafs := leafReviewers.Difference(stage1)
-	for stage1.Len()+stage2.Len() < minReviewers && unusedLeafs.Len() > 0 {
+	unusedLeafs := leafReviewers.Difference(reviewers.Set())
```

Comment:
Shouldn't this be `reviwers[0]`?

Reply:
In that case, yes, but by not assuming we only have one layer we future proof that code.